packages/java-edition/test/mcfunction/parser/argument.spec.ts

```diff
@@ -290,26 +305,32 @@ const RemoveExtraChildren = new Set([
 	'minecraft:score_holder',
 ])
 
-const project = mockProjectData()
-
-json.initialize(project)
-nbt.initialize(project)
-
-const { meta } = project
-
 describe('mcfunction argument parser', () => {
 	for (const [parserName, cases] of Object.entries(Suites)) {
 		describe(parserName, () => {
-			for (const { content, properties } of cases) {
+			for (const { content, properties, version } of cases) {
 				const treeNode: ArgumentTreeNode = {
 					type: 'argument',
 					parser: parserName as any,
 					properties,
 				}
+				const ctx = typeof version !== 'undefined'
+					? { loadedVersion: version }
+					: void 0
```

Comment:
Yeah the `void 0` threw me off here. Usually you only see it in minified code because it's shorter than `undefined`.

Reply:
Maybe the better thing here is to undo the `loadedVersion: '1.15'` in `core` (because it should only be in `java-edition`, and use `{ loadedVersion: version ?? '1.15' }` in this file?